docs/jupyterlite.rst

```diff
@@ -1,30 +1,29 @@
 Try out now with JupyterLite_
 -----------------------------
 
-.. admonition:: To try pyhf code in the browser with the Pyolite JupyterLite kernel:
+.. admonition:: Click below to try pyhf in the browser:
     :class: dropdown
 
      #. Type (or copy and paste) code in the input cell.
      #. To execute the code, press ``Shift + Enter`` or click on the (Run) ▶ button in the toolbar.
```

Comment:
At the moment with `replite` there is no toolbar anymore. Might want to work on flipping over to `jupyterlite` explicitly.

Reply:
You probably want `retrolite`, which gives a single notebook (that's what we use on awkward-array.org)